e2e_test/src/coordinator_tests.rs

```diff
@@ -0,0 +1,379 @@
+        kill_process("cnosdb");
+    }
+
+    #[tokio::test]
+    async fn test_replica() {
+        // clean env
+        kill_process("cnosdb-meta");
+        kill_process("cnosdb");
+        // start cluster
+        let opt: Output = start_cluster().await;
+        assert!(opt.status.code() == Some(0));
+        // database option schema
+        let dboption = DatabaseOptions::new(
+            Duration::new("365"),
+            Some(2),
+            Duration::new("365"),
+            Some(2),
+            Some(Precision::NS),
+        );
+        let dbschema = DatabaseSchema::new_with_options("cnosdb", "db1", dboption);
+        let cli = client::MetaHttpClient::new("127.0.0.1:8901".to_string());
+
+        let req = command::WriteCommand::CreateDB(
+            "cluster_xxx".to_string(),
+            "cnosdb".to_string(),
+            dbschema,
+        );
+        // create db
+        let _rsp = cli
+            .write::<command::TenaneMetaDataResp>(&req)
+            .await
+            .unwrap();
+        let api = "http://127.0.0.1:8901/debug".to_string();
+        let output = Command::new("curl")
+            .args([&api])
+            .output()
+            .expect("failed to execute process");
+        assert!(output.status.code() == Some(0));
+        print!("output: {}", String::from_utf8_lossy(&output.stdout));
+        assert!(String::from_utf8_lossy(&output.stdout).contains("\"replica\":2"));
+        // todo: check replica
+        // clean env
+        kill_process("cnosdb-meta");
+        kill_process("cnosdb");
+    }
+
+    #[tokio::test]
+    async fn test_shard() {
+        // clean env
+        kill_process("cnosdb-meta");
+        kill_process("cnosdb");
+        // start cluster
+        let opt: Output = start_cluster().await;
+        assert!(opt.status.code() == Some(0));
+        // database option schema
+        let dboption = DatabaseOptions::new(
+            Duration::new("365"),
+            Some(2),
+            Duration::new("365"),
+            Some(2),
+            Some(Precision::NS),
+        );
+        let dbschema = DatabaseSchema::new_with_options("cnosdb", "db1", dboption);
+        let cli = client::MetaHttpClient::new("127.0.0.1:8901".to_string());
+
+        let req = command::WriteCommand::CreateDB(
+            "cluster_xxx".to_string(),
+            "cnosdb".to_string(),
+            dbschema,
+        );
+        // create db
+        let _rsp = cli
+            .write::<command::TenaneMetaDataResp>(&req)
+            .await
+            .unwrap();
+        let api = "http://127.0.0.1:8901/debug".to_string();
+        let output = Command::new("curl")
+            .args([&api])
+            .output()
+            .expect("failed to execute process");
+        assert!(output.status.code() == Some(0));
+        print!("output: {}", String::from_utf8_lossy(&output.stdout));
+        assert!(String::from_utf8_lossy(&output.stdout).contains("\"shard_num\":2"));
```

Comment:
In the meta data of a database, `replica` and `shard_num` are in one line:

```
/cluster_xxx/tenants/cnosdb/dbs/db1: {"tenant":"cnosdb","database":"db1","config":{"ttl":"365d","shard_num":2,"vnode_duration":"365d,"replica":2,"precision":"NS")}}
```

Why have you created so many test cases?